src/riscv-integration.adoc

```diff
@@ -1078,27 +1078,44 @@ memory addresses must match for the address to be valid:
 * For Sv48, bits [63:48] must equal bit 47
 * For Sv57, bits [63:57] must equal bit 56
 
-RISC-V permits that some CSRs, such as <<mtvec>> and <<mepc>> (see
-xref:CSR_exevectors[xrefstyle=short]), need not be able to hold all possible
-invalid addresses. Prior to writing these CSRs, implementations may convert an
+RISC-V permits that some CSRs holding addresses, such as <<mtvec>> and <<mepc>>
+(see xref:CSR_exevectors[xrefstyle=short]) and consequently <<pcc>>, need not be able to hold all possible
+invalid addresses. Prior to writing these registers, implementations may convert an
 invalid address into some other invalid address that the register is capable of
 holding. However, these registers hold capabilities in {cheri_base_ext_name}
 and the bounds encoding depends on the address value, so implementations must
 not convert invalid addresses to other arbitrary invalid addresses in an
 unrestricted manner.
 The following procedure must be used instead when writing a capability A to
-these CSRs:
+a CSR that cannot hold all invalid addresses:
 
-. If A's address cannot be held then convert it to another address that the CSR can
-hold
-. If conversion _was_ required, then A's tag is cleared if A is
-sealed or if the new address is not representable -- this is equivalent to the
-semantics of <<SCADDR>>
+. If A's address cannot be held, then convert it to another address that the
+register can hold.
+. If A's address is invalid and A does not have infinite bounds (see
```

Comment:
the first and second point are the same one - it cannot be held if it's invalid -it's the same concept - so better to combine these two

Reply:
Fixed